README.md

```diff
@@ -30,7 +30,7 @@ objects:
   data:
     hazelcast.xml: |-
       <?xml version="1.0" encoding="UTF-8"?>
-      <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.10.xsd"
+      <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.12.xsd"
```

Comment:
I just do not use versions then xml is validated by the latest xsd in the jar.

Reply:
3.11 for now. Let's change it to 3.12 when 3.12 is released.